### PR TITLE
Add image

### PR DESCRIPTION
**Pull request by mwarnck**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,6 +16,10 @@ <h1>Mirco Warnck</h1>
       <a href="#biography">Biography</a>
     </nav>
   </header>
+  <figure>
+  <img src="https://source.unsplash.com/random/600x300" alt="Random Bild">
+  <figcaption>Ein random Bild</figcaption>
+</figure>
```
Comment (PReichetanz) [inline index.html]: Looks good, but please check the indentation of these lines.

File: index.html [hunk 2]
```diff
@@ -16,6 +16,10 @@ <h1>Mirco Warnck</h1>
       <a href="#biography">Biography</a>
     </nav>
   </header>
+  <figure>
+  <img src="https://source.unsplash.com/random/600x300" alt="Random Bild">
```
Comment (PReichetanz) [inline index.html]: I think that "Random Bild" is not the best way of combining English and German here ;)

**Review by PReichetanz — COMMENTED** [review]
Nice work, I only have minor comments.

File: index.html [hunk 3]
```diff
@@ -16,6 +16,10 @@ <h1>Mirco Warnck</h1>
       <a href="#biography">Biography</a>
     </nav>
   </header>
+  <figure>
+  <img src="https://source.unsplash.com/random/600x300" alt="Random image">
+  <figcaption>A random image for illustration</figcaption>
```
Comment (badabam) [inline index.html:21]: Please indent those two lines as they are nested in `figure`

**Review by badabam — COMMENTED** [review]
Please update this code in your next branch (even if it is unrelated)

**Pull request merged**